src/app/auth/account/additional-components/account-flows-tab/account-flows-tab.component.ts

```diff
@@ -0,0 +1,149 @@
+import { ChangeDetectionStrategy, Component, Input, OnInit } from "@angular/core";
+import { Observable, combineLatest, map, switchMap, timer } from "rxjs";
+import { MaybeNull, MaybeUndefined } from "src/app/common/app.types";
+import {
+    CancelFlowArgs,
+    FilterByInitiatorEnum,
+    FlowsTableData,
+} from "src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.types";
+import { Dataset, FlowStatus, FlowSummaryDataFragment, InitiatorFilterInput } from "src/app/api/kamu.graphql.interface";
+import { AccountService } from "src/app/services/account.service";
+import { AccountTabs } from "../../account.constants";
+import ProjectLinks from "src/app/project-links";
+import { requireValue } from "src/app/common/app.helpers";
+import { environment } from "src/environments/environment";
+import { FlowsTableProcessingBaseComponent } from "src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table-processing-base.component";
+
+@Component({
+    selector: "app-account-flows-tab",
+    templateUrl: "./account-flows-tab.component.html",
+    styleUrls: ["./account-flows-tab.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AccountFlowsTabComponent extends FlowsTableProcessingBaseComponent implements OnInit {
+    @Input() accountName: string;
+    public nodes: FlowSummaryDataFragment[] = [];
+    public searchByDataset: MaybeNull<Dataset> = null;
+    public currentPage = 1;
```

Comment:
`currentPage` present in both components

Reply:
Moved to base class